Cargo.toml

```diff
@@ -27,13 +27,14 @@ dev = ["clippy"]
 nightly = ["libc", "spin/unstable"]
 push = ["hyper", "libc"]
 process = ["libc", "procinfo"]
+gen = []
```

Comment:
I would like to know why do we need to add a new feature? What will happen if user enables / disables this feature?

Reply:
> It seems the `protoc-rust` depend on the `protoc` program which need be installed before build the crate.

The feature provide to the library developer who need use the latest `protobuf` version and installed the `protoc` program, and it should be disabled to make more easier with pre-generated binding file.

Reply:
### Note

The crate has a pre-generated protobuf binding file for `protobuf` v1.6, if you need use the latest version of `protobuf`, you can generate the binding file on building with the `gen` feature after you [download](https://github.com/google/protobuf) and install the the protocol buffer compiler.

```toml
[dependencies.prometheus]
git = "https://github.com/pingcap/rust-prometheus.git"
features = ["gen]
```